posthog/hogql/printer.py

```diff
@@ -1100,6 +1102,23 @@ def visit_call(self, node: ast.Call):
                 args_count = len(node.args) - func_meta.passthrough_suffix_args_count
                 node_args, passthrough_suffix_args = node.args[:args_count], node.args[args_count:]
 
+                if node.name in SURVEY_FUNCTIONS:
+                    if node.name == "getSurveyResponse":
+                        question_index_obj = node_args[0]
+                        if not isinstance(question_index_obj, ast.Constant):
+                            raise QueryError("getSurveyResponse first argument must be a constant")
+                        if (
+                            not isinstance(question_index_obj.value, int | str)
+                            or not str(question_index_obj.value).lstrip("-").isdigit()
+                        ):
+                            raise QueryError("getSurveyResponse first argument must be a valid integer")
+                        question_id = (
+                            str(node_args[1].value)
+                            if len(node_args) > 1 and isinstance(node_args[1], ast.Constant)
+                            else None
+                        )
```

Comment:
```
second_arg = node_args[1] if len(node_args) > 1 else None
question_id = str(second_arg.value) if isinstance(second_arg, ast.Constant) else None
```

would do a single lookup and still readable

Reply:
i'm not too keen on the advantages of optimizing lookups like this. no meaningful perf or readability improvements imo. i'll change though since no strong opinion

Reply:
theres no 2nd lookup here, please double check.